backend/src/middleware/permissionsMiddleware.js

```diff
@@ -27,6 +27,12 @@ const onlyYourself = rule({
   return context.user.id === args.id
 })
 
+const isMyEmotion = rule({
+  cache: 'no_cache',
+})(async (parent, args, context, info) => {
+  return context.user.id === args.from.id
```

Comment:
Eww - implement your own custom resolvers and you don't have to check that anymore.

![eww](https://media.giphy.com/media/122pEb8wGOZviM/giphy.gif)

Reply:
good catch